Tasks/google-play-release/GooglePlay.js

```diff
@@ -89,10 +89,11 @@ currentEdit = currentEdit.then(function (res) {
 });
 
 if (shouldAttachMetadata) {
+    var metadataRootPath = tl.getInput("metadataRootPath", false);
```

Comment:
Oh. I should have changed the false here to true. That makes the task library force it to have a non empty value.

And it should not be minimatched. There should only ever be one root folder. We can look into doing something else another time if we see that it is needed.